public/app/features/explore/Wrapper.tsx

```diff
@@ -46,9 +66,15 @@ export class Wrapper extends Component<WrapperProps> {
   }
 }
 
+function mapStateToProps(state: StoreState) {
+  return {
+    navModel: getNavModel(state.navIndex, 'explore'),
+  };
+}
+
 const mapDispatchToProps = {
   resetExploreAction,
   richHistoryUpdatedAction,
 };
 
-export default connect(null, mapDispatchToProps)(Wrapper);
+export default connect(mapStateToProps, mapDispatchToProps)(Wrapper);
```

Comment:
This is an old way of setting up Redux connections refer to [this for the preferred way](https://github.com/grafana/grafana/blob/main/contribute/style-guides/redux.md#typing-of-connected-props).

Reply:
Thank you @hugohaggmark 🙌🏾  I will take a look :)

Reply:
I did the changes here as well https://github.com/grafana/grafana/pull/34651/commits/0e6990946b6134881f68061c97c19d295ff8976d#diff-06aad0593d46c656f9c92b1a6e517f30e9758c1f1eff827194d81389ee5c74f6R15